dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -217,7 +217,7 @@
             id="http2ConnectionIdleTimeout" required="false" type="String" ibm:type="duration(s)" min="0" default="0" />
```

Comment:
Can you remove the contributors in the copyright since we're touching this file for this PR?